src/js/components/bulkDownload/awards/filters/AwardTypeFilter.jsx

```diff
@@ -33,6 +33,7 @@ export default class AwardTypeFilter extends React.Component {
     render() {
         const isValid = (
             this.props.currentAwardTypes.contracts ||
+            this.props.currentAwardTypes.idv ||
```

Comment:
This should be removed because updates to the Download Center are out of scope for this story.